scripts/ci.sh

```diff
@@ -73,7 +73,7 @@ setup_mf() {
 setup_lint() {
     # binary will be $(go env GOBIN)/golangci-lint
     curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOBIN) $GOLANGCI_LINT_VERSION
-
```

Comment:
What about this https://golangci-lint.run/usage/linters/#gocritic ?

Reply:
switched to a config file in latest